donjo-app/libraries/Curly.php

```diff
@@ -0,0 +1,375 @@
+<?php
+/**
+ * Curly - Simple cURL Library for PHP 5.4+
```

Comment:
Kalau library ini adalah sama (di ambil dari) yg anda kelola di https://github.com/esyede/curly, mohon tambahkan catatan di file `catatan_singkat_library_yang_digunakan.txt`

Reply:
Resolved in ab2d8d6